src/python/Publisher/utils.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import division
+import os
+import time
+import logging
+import hashlib
+import subprocess
+
+from WMCore.Services.SiteDB.SiteDB import SiteDBJSON
+from WMCore.Credential.Proxy import Proxy
+
+__version__ = '1.0.3'
+
+def getHashLfn(lfn):
+    """
+    Provide a hashed lfn from an lfn.
+    """
+    return hashlib.sha224(lfn).hexdigest()
+
+def getFTServer(site, view, db, log):
```

Comment:
Are we using this? Do we stuill need a couch DB?

Reply:
mmm.. let me check, probably not... but I'll create a issue for me to clean up the utils file next.. in the mean while I think we may want to include this version and test the deployment

Reply:
I think at present the only usefulness of the hash is that the table of xfer requests e.g. in the crabmonitor UI has fields of constant (<100) length. Going for LFN's may make it a bit messy, while more clear. I am sure we can review this !